unit_tests/connector_builder/test_connector_builder_handler.py

```diff
@@ -140,6 +142,19 @@
             "type": "DeclarativeStream",
             "$parameters": _stream_options,
             "retriever": "#/definitions/retriever",
+            "incremental_sync": {
```

Comment:
Added to test that the state is properly emitted. I also had to add `updated_at` fields in the records to avoid things from breaking